React/game/src/app/components/Screen/Game/Board/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
```

Comment:
Subcomponents (components inside other components) like `Game` and `Board` should follow the folder structure `ParentComponent/components/ChildrenComponent` so in this case, it should be `Game/components/Board`